/*
üöÄ Exerc√≠cio 4
Para finalizar o teste da fun√ß√£o printMessage, 
voc√™ pode ainda criar um fluxo de exce√ß√£o dentro
 da fun√ß√£o para criar um erro caso seja enviado um
  par√¢metro que n√£o seja um objeto, ou que n√£o tenha
   a propriedade personagem como no exemplo abaixo.

// printMessage.js
let info = {
  personagem: 'Margarida',
  origem: 'Pato Donald',
  nota: 'Namorada do personagem principal nos quadrinhos do Pato Donald',
};

const printMessage = (characterInfo) => {
  if (!characterInfo || characterInfo.personagem === undefined) {
    throw new Error('objeto inv√°lido');
  }
  return ('Boas vindas, '+ characterInfo.personagem);
};

console.log(printMessage(info));

module.exports = { info, printMessage };

Agora que a fun√ß√£o possui um fluxo de exce√ß√£o, crie uma 
fun√ß√£o de teste para validar se a mensagem de erro √© lan√ßada caso
 a fun√ß√£o seja chamada com um objeto inv√°lido.
 */